hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenAirdropValidator.java

```diff
@@ -121,8 +121,10 @@ public void validateSemantics(
                         .filter(item -> item.amount() < 0)
                         .findFirst();
                 final var senderId = senderAccountAmount.orElseThrow().accountIDOrThrow();
-                final var senderAccount = accountStore.get(senderId);
-                validateTrue(senderAccount != null, INVALID_ACCOUNT_ID);
+
+                final var senderAccount =
```

Comment:
This is no longer a `test:` but a `fix:` (PR title).  (Also:  Good thing the test was in the test plan!)